src/rtctools_heat_network/heat_mixin.py

```diff
@@ -4358,13 +4366,15 @@ def __cumulative_investments_made_in_eur_path_constraints(self, ensemble_member)
 
             # Asset can be realized once the investments made equal the installation and
             # investment cost
+            capex_sym = 0.0
+            if self.variable_nominal(self._asset_installation_cost_map[asset]) > 1.0e2:
+                capex_sym = capex_sym + installation_cost_sym
+            if self.variable_nominal(self._asset_investment_cost_map[asset]) > 1.0e2:
```

Comment:
Why was the value 100 chosen?

Reply:
the 100 is because of the default value we initialize the nominal on in the pre() when we create the variables.